operations/compute_rolling_upgrades/Upgrade_Compute_Nodes_with_CRUS.md

```diff
@@ -90,23 +91,24 @@ Upgrade a set of compute nodes with the Compute Rolling Upgrade Service \(CRUS\)
     workload_manager_type = "slurm"
     ```
 
-    If successful, note the upgrade\_id in the returned data.
+1. Note the `upgrade_id` in the returned data of the previous command.
 
     ```bash
-    ncn-w001# export UPGRADE_ID=e0131663-dbee-47c2-aa5c-13fe9b110242
+    ncn# UPGRADE_ID=e0131663-dbee-47c2-aa5c-13fe9b110242
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node (assuming this goes with a command listed below